pages.fr/common/grep.md

```diff
@@ -0,0 +1,36 @@
+# grep
+
+> Recherche des motifs dans un texte.
+> Supportes des motifs simple et des expressions rationnelles.
```

Comment:
from my point of view, a better translation for `regular expressions` in french is `expressions regulières` because it is this expression I always used in french.
```suggestion
> Supportes des motifs simple et des expressions regulières.
```